Description: Hide start/stop progressbar as done in KDE trunk
 The patch stolen from Kubuntu
Origin: vendor, http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/lucid/kdepimlibs/lucid/download/head:/kubuntu_02_hide_akon-20100413181428-vr64cd6ivtkqbzhp-1/kubuntu_02_hide_akonadi_progressbar.patch
Bug-Kubuntu: https://launchpad.net/bugs/562420

--- a/akonadi/control.cpp
+++ b/akonadi/control.cpp
@@ -130,8 +130,8 @@ class Control::Private
 
 bool Control::Private::exec()
 {
-  if ( mProgressIndicator )
-    mProgressIndicator->show();
+//  if ( mProgressIndicator )
+//    mProgressIndicator->show();
 
   kDebug() << "Starting/Stopping Akonadi (using an event loop).";
   mEventLoop = new QEventLoop( mParent );
